.github/workflows/ci.yml

```diff
@@ -32,66 +36,30 @@ jobs:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-java@v3
-        with:
-          java-version: 11
-          distribution: 'temurin'
 
-      - name: Setup Gradle
-        uses: gradle/gradle-build-action@v2
-        with:
-          gradle-home-cache-cleanup: true
-
-      - name: Build Library
-        run: ./gradlew --continue --stacktrace :image-generation:build
-        env:
-          CI: true
-
-      - name: Build Build Logic
-        run: ./gradlew --continue --stacktrace :buildSrc:build
-        env:
-          CI: true
-
-      - name: Build Applications
-        run: ./gradlew --continue --stacktrace build
-        env:
-          CI: true
-
-      - name: Comment PR with build scan on failure
```

Comment:
No more build scan on failures?

Reply:
All builds still publish build scans and the gradle build action summary still provides all the links.

What got removed is the automatic comment to the PR. the way it was done didn't work well with composite actions, I'll see at reinstating this another way.

Reply:
I see, thanks for the explanation.